[IntelliJ Plugin] Add diagram view and editing support (#15399)
ballerina-platform/ballerina-lang: tool-plugins/intellij/getting-started/plugin-features/README.md

![alt text](images/DebugSupport.gif)

## Diagram View

> Ballerina’s underlying language semantics were designed by modeling how independent parties communicate via structured interactions. 

Reviewer: ```suggestion
The underlying language semantics of Ballerina were designed by modeling how independent parties communicate via structured interactions. 
```

Author: Fixed.

> Subsequently, every Ballerina program can be displayed as a sequence diagram of its flow with endpoints, including synchronous and asynchronous calls. 

Reviewer: ```suggestion
Subsequently, every Ballerina program can be displayed as a sequence diagram of its flow with endpoints including synchronous and asynchronous calls. 
```

Author: Fixed.

> To view the sequence diagram of a ballerina file press the diagram icon 

Reviewer: ```suggestion
To view the sequence diagram of a ballerina file, click the diagram icon 
```

Author: Changed.

( ![design view icon](https://raw.githubusercontent.com/ballerina-platform/ballerina-lang/2fd0bdd4e7d081adf23901ed65eca32623d81889/tool-plugins/vscode/docs/show-diagram-icon.png) ) 

> in top right corner.

Reviewer: ```suggestion
in the top right corner of the screen.
```

Author: Changed.

![alt text](images/DiagramView.gif)

## Import Modules On-the-fly

> This feature adds import declarations on the fly. All you need to do is selecting the module name from the lookup list and the module declaration will be added automatically.

Reviewer: ```suggestion
This feature adds import declarations on the fly. All you need to do is to select the module name from the lookup list and then the module declaration will be added automatically.
```

Author: Fixed.
